sql3/planner/executionplanner.go

```diff
@@ -154,12 +169,19 @@ func (p *ExecutionPlanner) analyzePlan(ctx context.Context, stmt parser.Statemen
 		return nil
 	case *parser.DropViewStatement:
 		return nil
+	case *parser.DropModelStatement:
```

Comment:
Looking at this, I'm struck by the asymmetry that we support both create and drop for models, but not for functions. Should we support drop function also?

Reply:
Yes, but functions are (very) experimental, so drop function can come later.